config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,9 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider               => 'AWS',
+    :aws_access_key_id      => ENV["aws_access_key_rmx"],
+    :aws_secret_access_key  => ENV["aws_access_token_rmx"],
+    :region                 => 'eu-central-1'
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.